app.js

```diff
@@ -11,60 +11,78 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+
 // set state for all of the character's catchphrases
+let phraseArray = ['Welcome to your super perfect and realistic character customizer'];
+
+displayCatchphrases();
```

Comment:
You're calling this function in catchphrase event listener already. You shouldn't need to call this function here.